### PR TITLE
【Hackathon 6th Fundable Projects 3 No.266】[fluid_ops] partial_recv 2

### PR DESCRIPTION
**Pull request by co63oc**
### PR Category

Operator Mechanism

### PR Types

Others 

### Description

partial_recv

### CODE REVIEW

File: paddle/phi/kernels/gpu/partial_recv_kernel.cu [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#include "paddle/phi/core/kernel_registry.h"
+#if defined(PADDLE_WITH_NCCL) || defined(PADDLE_WITH_RCCL)
+#include "paddle/phi/core/distributed/nccl_comm_context.h"
+#endif
+
+namespace phi {
+
+template <typename T, typename Context>
+void PartialRecvOpCUDAKernel(const Context& dev_ctx,
+                             int ring_id,
```
Comment (liym27) [inline paddle/phi/kernels/gpu/partial_recv_kernel.cu]: ring_id kernel中用不到，这里删掉。或者标记 UNUSED

File: paddle/phi/kernels/gpu/partial_recv_kernel.cu [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+#include "paddle/phi/core/distributed/utils.h"
+#include "paddle/phi/core/kernel_registry.h"
+#if defined(PADDLE_WITH_NCCL) || defined(PADDLE_WITH_RCCL)
+#include "paddle/phi/core/distributed/nccl_comm_context.h"
+#endif
+
+namespace phi {
+
+template <typename T, typename Context>
+void PartialRecvOpCUDAKernel(const Context& dev_ctx,
```
Comment (liym27) [inline paddle/phi/kernels/gpu/partial_recv_kernel.cu]: PartialRecvOpCUDAKernel 改成 PartialRecvKernel

File: paddle/phi/kernels/cpu/partial_recv_kernel.cc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/phi/core/kernel_registry.h"
+
+namespace phi {
+
+template <typename T, typename Context>
+void PartialRecvOpCPUKernel(const Context& dev_ctx UNUSED,
```
Comment (liym27) [inline paddle/phi/kernels/cpu/partial_recv_kernel.cc]: PartialRecvOpCPUKernel 改成 PartialRecvKernel

File: paddle/phi/kernels/gpu/partial_recv_kernel.cu (hunk 1 above)
Comment (co63oc) [reply paddle/phi/kernels/gpu/partial_recv_kernel.cu]: done

File: paddle/phi/kernels/cpu/partial_recv_kernel.cc (hunk 3 above)
Comment (co63oc) [reply paddle/phi/kernels/cpu/partial_recv_kernel.cc]: done

File: paddle/phi/kernels/gpu/partial_recv_kernel.cu (hunk 2 above)
Comment (co63oc) [reply paddle/phi/kernels/gpu/partial_recv_kernel.cu]: done

**Review by liym27 — APPROVED** [review]
LGTM

**Pull request merged**